src/index.ts

```diff
@@ -1,36 +1,58 @@
 import server from 'server'
 import { status, render, redirect, Reply } from 'server/reply'
 import { get, post } from 'server/router'
-import fundAccount, { FUNDABLE_AMOUNT } from './fundAccount'
 
-const rateLimiter: { [key: string]: Date } = {}
+import { Connection } from './shared/connections'
+import fundAccount from './fundAccount'
+
+const rateLimiter: {
+  [currency: string]: {
+    [key: string]: Date
+  }
+} = {}
 const limit = 1000 * 60 * 60 // 1 hour
```

Comment:
I would change the name of this constant to something more expressive and delete the comment.